autogluon/training/docker/0.2/py3/cu102/Dockerfile.gpu

```diff
@@ -0,0 +1,85 @@
+ARG CUDA_VER=10.2
+ARG LINUX_VER=ubuntu18.04
+ARG PYTHON_VERSION=3.7.10
+ARG PYTHON_VER=3.7
+ARG RAPIDS_VER=0.19
+ARG FROM_IMAGE=rapidsai/rapidsai-core
+
+FROM ${FROM_IMAGE}:${RAPIDS_VER}-cuda${CUDA_VER}-base-${LINUX_VER}-py${PYTHON_VER}
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG CUDA_VER
```

Comment:
nit: `ARG CUDA_VER=10.2` is defined on line 1

Reply:
https://github.com/aws/deep-learning-containers/pull/1202#discussion_r676941928